Geometry/HGCalCommonData/test/HGCalConvert.cpp

```diff
@@ -51,6 +51,12 @@ const int kHGCalPhiOffset = 0;
 const int kHGCalPhiMask = 0x1FF;
 const int kHGCalRingOffset = 9;
 const int kHGCalRingMask = 0x1FF;
+const int kHGCalFactor = 10;
```

Comment:
rather than duplicating the functionality of `HGCalProperty`, could that class/namespace be used directly in this test code?

Reply:
Done